packages/@aws-cdk/aws-lambda/lib/runtime.ts

```diff
@@ -161,24 +161,25 @@ export class Runtime {
 
   /**
    * The .NET Core 1.0 runtime (dotnetcore1.0)
-   * Legacy runtime no longer supported by AWS Lambda. Migrate to the latest .NET Core runtime.
+   * @deprecated Legacy runtime no longer supported by AWS Lambda. Migrate to the latest .NET Core runtime.
    */
   public static readonly DOTNET_CORE_1 = new Runtime('dotnetcore1.0', RuntimeFamily.DOTNET_CORE);
 
   /**
    * The .NET Core 2.0 runtime (dotnetcore2.0)
-   * Legacy runtime no longer supported by AWS Lambda. Migrate to the latest .NET Core runtime.
+   * @deprecated Legacy runtime no longer supported by AWS Lambda. Migrate to the latest .NET Core runtime.
    */
   public static readonly DOTNET_CORE_2 = new Runtime('dotnetcore2.0', RuntimeFamily.DOTNET_CORE);
 
   /**
    * The .NET Core 2.1 runtime (dotnetcore2.1)
-   * Legacy runtime no longer supported by AWS Lambda. Migrate to the latest .NET Core runtime.
+   * @deprecated Legacy runtime no longer supported by AWS Lambda. Migrate to the latest .NET Core runtime.
    */
   public static readonly DOTNET_CORE_2_1 = new Runtime('dotnetcore2.1', RuntimeFamily.DOTNET_CORE);
 
   /**
    * The .NET Core 3.1 runtime (dotnetcore3.1)
+   * @deprecated Legacy runtime no longer supported by AWS Lambda. Migrate to the latest .NET Core runtime.
```

Comment:
I don't see 3.1 in the list of deprecated runtimes in the docs that you linked.

Reply:
![image](https://user-images.githubusercontent.com/46012524/202710672-6896d27c-5192-4410-8b43-2091fa83c40b.jpeg)

hello sir, I think it will be deprecated after marking the next 6 months like nodejs12, if I misunderstood, I am sorry, I will close this PR, thanks.

Reply:
@neilkuan ah sorry! I didn't scroll over to see the deprecation there! I can't reopen this since the branch has been deleted, but if you recreate it i'll approve.